orc8r/cloud/go/services/state/servicers/southbound/helpers.go

```diff
@@ -0,0 +1,77 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package servicers
```

Comment:
If we're gonna convert this file's contents to public, let's put it in like orc8r/cloud/go/services/state/conversion.go, or something like that (and maybe the err goes in .../error.go)